bicep-templates/DSC/AvdFunctions.ps1

```diff
@@ -0,0 +1,9 @@
+function GetAvdSessionHostName {
```

Comment:
are there any changes in the dsc folder or is it just copy pasted?

Reply:
I just copy pasted the DSC folder as I didn't see anything needing to be changed

Reply:
Maybe we should refactor the dsc folder out and use the same one for both arm and bicep templates